### ISSUE TITLE
Verify Multiple Local Unit Proxy Operation

### ISSUE THREAD

**Issue by kb1lqd** [FaradayRF/Faraday-Software#32]
Verify that multiple local units can be independently communicated with using the new single proxy interface. This will greatly ease development resources and allow much more simple single user experimentation. Prior proxy interaction required two independently configured proxy's to operate and were hard to manage.

**Comment by kb1lqd** [comment]
I've configured the proxy "faraday.ini" file for two unit operation on ports 3 and 12 locally. I've assigned:
- Unit 0: KB1LQD-1
- Unit 1: KB1LQC-1

Although an unfortunately these are static and won't follow the units ACTUAL configuration. For example, if KB1LQD-1 was reconfigured to KB1LQD-4 or KB1LQC-3 the units proxy address would remain KB1LQD-1. In the future we should update this to follow the units current callsign as this would be fairly simple to just query the current unit configuration and update the INI file accordingly. The COM port is the important static setting here.

![image](https://cloud.githubusercontent.com/assets/339144/20248541/1b92c55a-a99b-11e6-8867-f1eb7e4d5453.png)

Running "Tutorial 1b" which is an example of command querying the local unit(s) I am able to determine the unit being addressed. Note that it is important that the tutorial "flush" the old data from the buffer to remove any RF received telemetry packets as they are also present and units configured to beacon every few seconds. 

**KB1LQD-1**

Querying telemetry packet #3 (standard unit telemetry) obtained this configuration from the unit:
![image](https://cloud.githubusercontent.com/assets/339144/20248563/8a4bae08-a99b-11e6-97d8-deab8517602d.png)

It is clearly KB1LQC-1 which is OK (albeit confusing) because I randomly hooked these units up to the computer. This highlights the need to create dynamic unit address update.

This setting was repeatable on multiply queries.

**KB1LQC-1**
Adressing the query to "KB1LQC-1" returned different data that showed the unit "KB1LQD-20" was connected to this port. Again, highlighting the need for dynamically updating address but clearly it's working!

![image](https://cloud.githubusercontent.com/assets/339144/20248579/e5f3e842-a99b-11e6-9c0d-c300418c7885.png)

**Conclusion**
Multiple local unit proxy is working.

**Comment by kb1lqd** [comment]
I wanted to make sure unit configuration was operating correctly as well so I started the "device configuration" program alongside the proxy:

![image](https://cloud.githubusercontent.com/assets/339144/20248587/2c557594-a99c-11e6-9720-c4ca47a02c94.png)

This should operate cleanly with BOTH local units as addressed during commanding.

NOTE: I fixed a bug with the device configuration program that used hardcoded callsign and unit ID. This was causing confusion as one unit worked properly but the other seemingly received the other units data after programming. Fixed now.

**KB1LQC-1**

I updated the unit from it's current configuration to -1 to -2 as a test. I did not restart any proxy or device configuration program but just addressed the two units respectively. Note that the units local "address" remains KB1LQC-1 per static proxy addressing.

![image](https://cloud.githubusercontent.com/assets/339144/20248867/ca73c6d6-a9a1-11e6-9c22-9c5b9009865e.png)

**KB1LQD-1**

I updated the unit from it's current configuration to -1 to -2 as a test. I did not restart any proxy or device configuration program but just addressed the two units respectively. Note that the units local "address" remains KB1LQD-1 per static proxy addressing.

![image](https://cloud.githubusercontent.com/assets/339144/20248877/0aca6730-a9a2-11e6-8050-5f8dba0ed79a.png)
